railties/lib/rails/commands/server/server_command.rb

```diff
@@ -98,7 +98,7 @@ def log_to_stdout
       end
 
       def restart_command
-        "bin/rails server #{ARGV.join(' ')}"
+        "rails server #{ARGV.join(' ')}"
```

Comment:
While we might _talk_ about running `rails`, I think we're better off still going straight to `bin/rails` when we're the ones doing the running. That saves us a few milliseconds of indirection to get back to the same place, and avoids trouble if anyone has an unusual PATH setup -- I'm happy to let such people handle mental translation of `rails` to whatever special command they need, but avoidably breaking because of it seems less good.